mmv1/third_party/terraform/services/resourcemanager/ephemeral_google_service_account_id_token_test.go

```diff
@@ -0,0 +1,107 @@
+package resourcemanager_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccEphemeralServiceAccountIdToken_basic(t *testing.T) {
+	t.Parallel()
+
+	serviceAccount := envvar.GetTestServiceAccountFromEnv(t)
+	targetServiceAccountEmail := acctest.BootstrapServiceAccount(t, "idtoken", serviceAccount)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time": {},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccEphemeralServiceAccountIdToken_basic(targetServiceAccountEmail),
+			},
+		},
+	})
+}
+
+func TestAccEphemeralServiceAccountIdToken_withDelegates(t *testing.T) {
+	t.Parallel()
+
+	initialServiceAccount := envvar.GetTestServiceAccountFromEnv(t)
+	delegateServiceAccountEmailOne := acctest.BootstrapServiceAccount(t, "delegate1", initialServiceAccount)          // SA_2
+	delegateServiceAccountEmailTwo := acctest.BootstrapServiceAccount(t, "delegate2", delegateServiceAccountEmailOne) // SA_3
+	targetServiceAccountEmail := acctest.BootstrapServiceAccount(t, "target", delegateServiceAccountEmailTwo)         // SA_4
```

Comment:
We need to change the testId arguments passed in here, according to [the comment about how to use acctest.BootstrapServiceAccount here](https://github.com/hashicorp/terraform-provider-google/blob/6c9f75fb1fa0b42d6a3651bdc11920f7dda22412/google/acctest/bootstrap_test_utils.go#L221-L223). The testIds "delegate1", "delegate2", "target" are used in a previous PR of yours, so different values need to be used here. What value to use is arbitrary, but here are some suggestions

```suggestion
	delegateServiceAccountEmailOne := acctest.BootstrapServiceAccount(t, "id-delegate1", initialServiceAccount)          // SA_2
	delegateServiceAccountEmailTwo := acctest.BootstrapServiceAccount(t, id-delegate2", delegateServiceAccountEmailOne) // SA_3
	targetServiceAccountEmail := acctest.BootstrapServiceAccount(t, "id-target", delegateServiceAccountEmailTwo)         // SA_4
```